### DISCUSSION TITLE
Text within text

### DISCUSSION THREAD

**Discussion by TomasSalas** [diegomura/react-pdf#2751] [General]
`<Text> Text <Text Style={{fontWeight: ‘bold’}} > Text in bola </Text></Text>`

the text that has a bold style does not change within the text... and I leave the two texts isolated if it takes the effect any help on this?

**Comment by TomasSalas** [comment]
<img width="1112" alt="Captura de pantalla 2024-05-11 a la(s) 11 23 48 p  m" src="https://github.com/diegomura/react-pdf/assets/57121296/843cc7bf-7ce7-4df5-80c9-bdf6159c4e38">


this is example not found FontWeight bold

**Comment by TomasSalas** [comment]
@diegomura

**Comment by lecstor** [comment]
@TomasSalas I believe this should be fixed by https://github.com/diegomura/react-pdf/pull/2747
